homeassistant/components/powerwall/sensor.py

```diff
@@ -137,6 +137,12 @@ async def async_setup_entry(
             for description in POWERWALL_INSTANT_SENSORS
         )
 
+    for battery in data.batteries.values():
+        for description in BATTERY_INSTANT_SENSORS:
+            entities.append(
```

Comment:
It would be nicer to keep using extend as above so everything here works the same